StRoot/StRefMultCorr/StBadRunChecker.h

```diff
@@ -0,0 +1,50 @@
+#ifndef __StBadRunChecker_h__
+#define __StBadRunChecker_h__
+
+#include <vector>
+#include <map>
+#include "TString.h"
+
+#include "BadRunSubSys.h"
+//______________________________________________________________________________
+class StBadRunChecker {
+ public:
+  StBadRunChecker(TString Run="run19",TString CollisionMode="col" ,TString RunEnergy="19.6",TString Species="auau");
+  virtual ~StBadRunChecker(); /// Default destructor
+ 
+  // Bad run rejection
+  Bool_t isInjection(const Int_t RunId) ;
+  Bool_t isBadRunTPC(const Int_t RunId) ;
+  Bool_t isBadRunbTOFStatus(const Int_t RunId) ;
+  Bool_t isBadRunbTOFPID(const Int_t RunId) ;
+  Bool_t isBadRuneTOF(const Int_t RunId) ;
+  Bool_t isBadRunEPD(const Int_t RunId) ;
+  Bool_t isBadRunVPD(const Int_t RunId) ;
+  Bool_t isBadRunBEMCStatus(const Int_t RunId) ;
+  Bool_t isBadRunBEMCPID(const Int_t RunId) ;
+  Bool_t isBadRunBEMCTrigger(const Int_t RunId) ;
+  Bool_t isBadRunMTD(const Int_t RunId) ;
+  Bool_t isBadRunAnalysis(const Int_t RunId) ; 
+  Bool_t isBadRun(const Int_t RunId,TString mSys) ; 
```

Comment:
Would be good to add a comment that "mSys" could be for a single sub-system or a combination of different sub-systems, such as "injection" or "tpc_etof_vpd", etc.